tests/test_core.py

```diff
@@ -744,6 +744,13 @@ def test_interpolate_with_list(test_df_year):
     npt.assert_allclose(obs, [3, 4, 1.5, 2, 4, 5])
 
 
+def test_interpolate_with_numpy_list(test_df_year):
+    test_df_year.interpolate(np.r_[2007 : 2008 + 1], inplace=True)
+    obs = test_df_year.filter(year=[2007, 2008])._data.reset_index(drop=True)
+    exp = pd.Series([3, 4, 1.5, 2, 4, 5], name="value")
+    pd.testing.assert_series_equal(obs, exp)
```

Comment:
The other tests in that section use the following, any particular reason not to follow that convention?
```suggestion
    obs = test_df_year.filter(year=[2007, 2008])._data.values
    npt.assert_allclose(obs, [3, 4, 1.5, 2, 4, 5])
```